src/commands/registerCommands.ts

```diff
@@ -159,6 +159,12 @@ for (const label of bindingTypes.keys()) {
   bindingTypeOptions.push({ "label": label });
 }
 
+const bindingTypeOptionsForRelaxNG: QuickPickItem[] = [];
+for (const entry of bindingTypes.entries()) {
+  if (entry[1] != 'standard') {
```

Comment:
it's generally a good idea to use `!==` instead of `!=` (it won't make a difference in the logic in this case, though).

(apparently the eslint rule to check this, `"eqeqeq"`, is off by default, which is an odd decision)

Reply:
I missed that, thanks for reporting that.

fixed